examples/quantization/rep_qat.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import os
+import sys
+
+CURRENT_PATH = os.path.abspath(os.path.dirname(__file__))
+
+sys.path.insert(1, os.path.join(CURRENT_PATH, '../../'))
+
+import functools
+from distutils.version import LooseVersion, StrictVersion
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.quantization as torch_q
+
+from examples.models.cifar10.mobilenet import DEFAULT_STATE_DICT, Mobilenet
+from tinynn.converter import TFLiteConverter
+from tinynn.graph.quantization.quantizer import QATQuantizer, PostQuantizer, load_processed_ptq_rules
+from tinynn.graph.tracer import model_tracer, trace
+from tinynn.util.cifar10 import get_dataloader, train_one_epoch, validate, calibrate
+from tinynn.util.train_util import DLContext, get_device, train
+
+from tinynn.util.cross_layer_equal import weight_euqal_
+from tinynn.util.bn_rebuild import model_add_bn
+
+
+def main_worker(args):
+    model = Mobilenet()
+    model.load_state_dict(torch.load(DEFAULT_STATE_DICT))
+
+    # Provide a viable input for the model
+    dummy_input = torch.rand((1, 3, 224, 224))
+
+    # We use BN_folded MobileNetv1 to simulate reparameterized MobileOne.
+    # You can also directly use the reparameterized model of MobileOne(or other rep-model).
+    fused_model, _ = get_fused_model(dummy_input, model)
+
+    # Do CLE, if qat init precision is not ideal, try to do twice CLE or lower threshold.
+    with torch.no_grad():
+        weight_euqal_(fused_model, dummy_input, threshold=1000)
```

Comment:
The internal interface should not be exposed to the outside, you can encapsulate a more understandable interface, such as `cross_layer_equal`